spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -584,26 +584,15 @@ implementations.
 
 ==== Rabbit MQ Binder properties
 
-The binder supports the all Spring Boot properties for Rabbit MQ configuration.
+By default, the binder uses the Spring Boot `ConnectionFactory` and therefore it supports all the Spring Boot configuration options for Rabbit MQ.
+For reference, consult the [Spring Boot documentation](http://docs.spring.io/spring-boot/docs/1.3.3.RELEASE/reference/htmlsingle/#common-application-properties). Rabbit MQ configuration options use the `spring.rabbitmq` prefix.
```

Comment:
Better to use the version-less symlink

http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/

will fix on merge